net/crowdsec-firewall-bouncer/files/crowdsec-firewall-bouncer.defaults

```diff
@@ -1,22 +1,48 @@
 #!/bin/sh
 
-CONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+CSCLI=/usr/bin/cscli
+CFG_FILE=/etc/crowdsec/config.yaml
+CSFB_CUSTOMCONFIG=/etc/crowdsec/bouncers/crowdsec-firewall-bouncer.yaml
+CSFNAME=crowdsec-firewall-bouncer
```

Comment:
This is interesting as well. Why don't you use full path?

Reply:
CSFNAME is not a path, it is a string value used to named the bouncer at registration and to test bouncer status.